packages/xchain-doge/src/client.ts

```diff
@@ -161,8 +161,13 @@ class Client extends UTXOClient {
    * @returns {TxHash} The transaction hash.
    */
   async transfer(params: TxParams & { feeRate?: FeeRate }): Promise<TxHash> {
-    const feeRate = params.feeRate || (await this.getSuggestedFeeRate())
-    checkFeeBounds(this.feeBounds, feeRate)
+    let feeRate: number
+    if (params.feeRate) {
+      feeRate = params.feeRate
+      checkFeeBounds(this.feeBounds, feeRate)
+    } else {
+      feeRate = await this.getSuggestedFeeRate()
+    }
```

Comment:
We shouldn't bypass `checkFeeBounds` functionality. If the `LOWER_FEE_BOUND` is too much, update it with a lower quantity.

If the DOGE network does not recommend less than that fee_rate then force a minimum of LOWER_FEE_BOUND.